tdrs-backend/reports/README.md

```diff
@@ -0,0 +1 @@
+# OWASP ZAP Scanner Reports
```

Comment:
These reports won't be available in the repository, but I needed to add a file to the directory to get it in the repository.

Reply:
As an alternative you could put a `.gitignore` file in these directories with the following contents:
```
*
*/
!.gitignore
```
This way if someone runs the `zap-scanner.sh` script locally they don't need to worry about accidentally committing the report file.

Reply:
That's a good idea. I'll do that.